### DISCUSSION TITLE
Where is SSC inventory data stored?

### DISCUSSION THREAD

**Discussion by zaneshaw** [Pryaxis/TShock#2066] [Q&A]
I accidentally wiped the log folder (default "tshock") and I was wondering if I also wiped my SSC data along with it.
Also, I don't know if files like config.json, motd.txt and rules.txt are supposed to be in there or not. They are also in the dir that I created for the server before setting it up which includes "1405" and "ServerPlugins".

**Comment by Kojirremer** [comment]
Yep, if you deleted that folder, you deleted every single config file and database generated by TShock and its plugins, which also includes tshock.sqlite, the database containing pretty much every important thing that TShock keeps track of, including registered users and their SSC inventories.

And yes, those files are meant to be there. IMO, the logs are what is more out of place on that folder than anything else, since that folder keeps configs and databases.

**Reply by zaneshaw** [reply]
Oh well, that sucks. Also, do you know why every time I run the server, it generates another "tshock" folder inside the already existing "tshock" folder.
My dir: /home/user/tserver/tshock/**tshock**

**Reply by Kojirremer** [reply]
> Oh well, that sucks. Also, do you know why every time I run the server, it generates another "tshock" folder inside the already existing "tshock" folder.

There is always going to be a tshock (lowercase) folder within the root TShock folder. You should respect that structure and not move files from how they are generated. The TShock executable will always search for its important files within the tshock folder, and in case it doesn't find any, will create both the folder and files.

**Reply by zaneshaw** [reply]
No I mean there are duplicate config files and such. Both the bold dirs contain the same files.
/home/user/tserver/**tshock**/**tshock**

**Reply by Kojirremer** [reply]
I'm assuming that's because you moved the root TShock files to the lowercase tshock folder, turning it into the new root TShock folder containing the executable. If that's the case, then my point about respecting how the files are generated remain (IE Don't move things from how they generate after launch).
If you are absolutely sure you didn't move the executable and accompanying files to the "tshock" config folder, then no idea what your problem is, sorry.

**Reply by zaneshaw** [reply]
Since I deleted the SQLite stuff and the world is stored elsewhere, do you think I would be better off just starting the server again in a fresh dir with the latest release? Also thanks for the help! 👍

**Reply by Kojirremer** [reply]
Yep. Anything you had set up was deleted with the "tshock" subfolder, so starting from scratch is probably the easier way right now.
